src/Helpers.php

```diff
@@ -252,16 +252,16 @@ public static function getOpenAPIAttributeTagsByScope(ClassMethod|Class_|Node $n
 							if ($item?->value instanceof String_) {
 								$tag = $item->value->value;
 								$pattern = '/^[0-9a-zA-Z_-]+$/';
-								if (!preg_match($pattern, $tag)) {
+								if (in_array(preg_match($pattern, $tag), [0, false], true)) {
```

Comment:
Do we mean  !== false?
or really also 0?

Reply:
False is failure (I guess invalid regex?) and 0 means no match, so this check is correct